src/polyfills/custom-elements.js

```diff
@@ -455,15 +463,28 @@ class Registry {
     this.query_ = name;
 
     // The first registered name starts the mutation observer.
-    const observer = new this.win_.MutationObserver(records => {
+    this.mutationObserver_ = new this.win_.MutationObserver(records => {
```

Comment:
When will the callback function be invoked now that we have `sync()`?

Reply:
Hopefully never, it's there just in case I've screwed up.